### DISCUSSION TITLE
Use icons for the prompt about quitting

### DISCUSSION THREAD

**Discussion by GrimPixel** [IndiePython/nodezator#63] [Feature Request]
The texts require some reading, so I think icons can help:
* Quit without saving ![](https://upload.wikimedia.org/wikipedia/commons/thumb/e/ef/ISO_7010_E016.svg/240px-ISO_7010_E016.svg.png)
* Save & Quit ![](https://upload.wikimedia.org/wikipedia/commons/thumb/2/27/ISO_7010_M030%3B_mandatory%3B_place_trash_in_the_bin.svg/240px-ISO_7010_M030%3B_mandatory%3B_place_trash_in_the_bin.svg.png)
* Cancel closing app ![](https://upload.wikimedia.org/wikipedia/commons/thumb/3/3c/ISO_7010_P070.svg/240px-ISO_7010_P070.svg.png)

**Comment by GrimPixel** [comment]
I think “save & quit” can be removed, because it is considered a bad habit to save at the prompt of closing.

**Comment by KennedyRichard** [comment]
Thank you for the suggestions. You are right, icons will help. There are actually a lot of labels/prompts in the app that lack icons to help make the actions more clear or more easily recognizable. When implementing this, I'll check the internet for commonly used icons for such actions, in order to use icons that are more easy to associate with the intended actions.

I also think you are right regarding the **Save and Quit** options. It is both simpler and better to give users the option to either quit or cancel quitting.
